README.md

```diff
@@ -146,6 +146,27 @@ subdirs:
   - win-64
 ```
 
+### Append to Existing Mirror
+
+When trying to add new packages to existing mirror, the `--append` flag can be used to prevent deletion of existing packages (default behavior). The repodata will be rewritten with this flag enabled.
+
+**Notes**
+
+- The CLI option will override the YAML option.
+- In append mode the tool skips deletions and repodata is written for the union of previously present files plus the files added in the current run.
```

Comment:
i'm not sure i follow here. how is this different from what conda-mirror does already?

conda-mirror already doesn't delete previously available files but just adds new packages in subsequent runs

Reply:
okay so this seems more like a memory issue. i can reproduce this, need to figure out what's going on.